trade-finance-manager-ui/server/controllers/case/amendments/validation/amendmentIsUsingFacilityEndDate.validate.test.js

```diff
@@ -0,0 +1,47 @@
+import { isUsingFacilityEndDateValidation } from './amendmentIsUsingFacilityEndDate.validate';
+
+describe('amendmentBankDecisionValidation()', () => {
+  it('should return an error if no choice is selected', () => {
+    const result = isUsingFacilityEndDateValidation(null);
+
+    const expected = {
```

Comment:
This return type is really odd but it looks like this is what the rest of the files in this directory do